packages/kbn-mock-idp-plugin/server/index.ts

```diff
@@ -17,6 +17,7 @@ export const config = {
     // The plugin should only be enabled in Serverless.
     enabled: offeringBasedSchema({
       serverless: schema.boolean({ defaultValue: true }),
+      traditional: schema.boolean({ defaultValue: false }),
```

Comment:
What is this about?

Reply:
it generally enables plugin in stateful: `traditional` stands for stateful in schema, I don't know why it was called like that. Default value is `false` because we need that plugin only when Kibana is configured with SAML auth and it is not always like that.